js/globals.ts

```diff
@@ -31,6 +14,16 @@ export const globalEval = eval;
 // TODO The underscore is because it's conflicting with @types/node.
 export const window = globalEval("this");
 
+// The built-in libdeno functions are moved out of the global variable.
+type MessageCallback = (msg: Uint8Array) => void;
+interface Libdeno {
```

Comment:
I am surprised you aren't getting an error here when the library files are being generated, as this interface isn't exported but an export of this module (`libdeno`) refers to it.

Reply:
Ignore me, it gets emitted in the declaration file, so TypeScript can see it.